lnbits/static/js/wallet.js

```diff
@@ -90,6 +90,7 @@ new Vue({
   mixins: [windowMixin],
   data: function () {
     return {
+      origin: window.location.origin,
```

Comment:
I don't think this is correct - when LNbits is deployed to https://foobar.example/lnbits then `window.location.origin` returns just `https://foobar.example` and not `https://foobar.example/lnbits` - or is this type of deployment unsupported?